lib/PuppeteerSharp/Page.cs

```diff
@@ -644,7 +644,14 @@ public Task ExposeFunctionAsync<TResult>(string name, Func<TResult> puppeteerFun
         public async Task EmulateMediaAsync(MediaType media)
             => await Client.SendAsync("Emulation.setEmulatedMedia", new { media });
 
-        public async Task SetViewport(ViewPortOptions viewport)
+        /// <summary>
+        /// Sets the viewport.
+        /// In the case of multiple pages in a single browser, each page can have its own viewport size.
+        /// NOTE in certain cases, setting viewport will reload the page in order to set the isMobile or hasTouch properties.
+        /// </summary>
+        /// <returns>The viewport task.</returns>
+        /// <param name="viewport">Viewport options.</param>
+        public async Task SetViewportAsync(ViewPortOptions viewport)
```

Comment:
maybe rename to `SetViewPortAsync`?

Reply:
The method is `setViewport` on Puppeteer.